.golangci.yml

```diff
@@ -33,6 +34,12 @@ linters-settings:
       alias: mesh_proto
     - pkg: github.com/kumahq/kuma/pkg/util/proto
       alias: util_proto
+  gomodguard:
+    blocked:
+      modules:
+      - github.com/go-errors/errors:
```

Comment:
what about just `errors` module?

Reply:
We could, if there's consensus. Though it would prevent adopting APIs like `errors.Is`.

Reply:
wait, do we want to block the standard `errors` module? why?

Reply:
> wait, do we want to block the standard `errors` module? why?

It is not blocked in this PR. The question was whether we ought to block it.

Reply:
why would we block it? We use `errors.New` all the time

Reply:
`errors.New` is a function from `pkg/errors`

Reply:
Yeh, we use both versions.